docs/App.fs

```diff
@@ -23,20 +22,39 @@ type Highlight =
     static member inline highlight (properties: IReactProperty list) =
         Interop.reactApi.createElement(importDefault "react-highlight", createObj !!properties)
 
-type State = {
-    CurrentPath : string list
-}
+type State = 
+    { CurrentPath : string list
+      CurrentTab: string list }
 
-let init() = { CurrentPath = [ ] }, Cmd.none
+let init () =
+    let path = 
+        match document.URL.Split('#') with
```

Comment:
As of the latest version of Feliz.Router, you can simply ask for the current url in which the page was initialized using the `Router.currentUrl()` function